SPECS/qlogic-netxtreme2-alt.spec

```diff
@@ -26,8 +24,9 @@ Group: System Environment/Kernel
 Requires: %{name}-%{modules_package} = %{version}-%{release}
 #Source taken from http://ldriver.qlogic.com/driver-srpms/netxtreme2/netxtreme2-7.14.46-1.rhel7u2.src.rpm
 Source: %{name_orig}-%{version}.tar.gz
+Patch0: 0001-install-into-versioned-dir.patch 
```

Comment:
See https://xcp-ng.org/forum/topic/183/git-repository-structure-and-conventions-for-rpms for the naming conventions of patches (I should move this document to the wiki)